lib/common.css

```diff
@@ -197,3 +197,13 @@ button.goog-buttonset-default:hover {
   color: <<FORE_SECONDARY>>;
 }
 /** Update End **/
+
+/** Fixes Playlist button **/
+.material .sj-play-button-0 #buttonContent.sj-play-button {
+    background: <<FORE_SECONDARY>>;
+}
+
+.material .sj-play-button-0 #pulse.sj-play-button {
+    background: <<FORE_SECONDARY>>;
+    opacity: .3;
```

Comment:
After Testing this it doesn't seem to work? http://puu.sh/rNbL9/3d54118a7f.png

Although i may have inputted it in wrong:
`.material .sj-play-button-0 #pulse.sj-play-button {
    background: <<ALPHA 0.3>><<FORE_SECONDARY>><<ALPHA>>;
}`